skimage/transform/_warps.py

```diff
@@ -142,6 +146,18 @@ def resize(
     symmetric, the result would be [0, 1, 2, 2, 1, 0, 0], while for reflect it
     would be [0, 1, 2, 1, 0, 1, 2].
 
+    `resize` uses interpolation. Unless the interpolation method is nearest-neighbor
+    (``order==0``), the algorithm will generate output values as a weighted average
```

Comment:
```suggestion
    (``order==0``), the algorithm will generate output values as weighted averages
```

Reply:
I intended for this to be understood as "following a weighted average procedure", so "weighted average" did not refer to an individual numerical result. But sure, this is less likely to raise eyebrows.

> Unrelated to this PR but

This was intended to be a very targeted PR to close gh-7524. I'd rather not expand the scope, if that's alright.